### ISSUE TITLE
General export seems to corrupt the backup

### ISSUE THREAD

**Issue by cmburg** [codinguser/gnucash-android#734]
#### Steps to reproduce the behaviour
    Transactions->Export
    Save As... (file in local device), Format (zip? -the standard setting)
    Export
    Open...

    Choose file from device.
    => 'An error occurred while importing the GnuCash accounts At line 1, column 2: not well-formed (invalid token)'

The error is both the same even when I try the zip or the (unzipped qif)

#### Expected behaviour
import the books
#### Actual behaviour
as described before
#### Software specifications
* GnuCash Android version:2.2.0
* System Android version:7.1.x (LineageOS)
* Device type:Nexus 5x

**Comment by Goli4thus** [comment]
Recently updated to Android 7.1.x (LineageOS) as well. 
Just tried to export for the first time since that switch. Same problem as OP.

But I think I might have a clue on why it's not working:
After the export via `Save As...` one get's back to Gnucash-android. Below the `Save As...` selection the path to which was exported to is shown.
Notice the beginning of the path: `content://com.android.externalstorage.......`

I recently had problems with KeepassDroid saving changes to its database.
Back then I searched around until I found this: https://github.com/bpellin/keepassdroid/issues/75
As taylormarshall states in his third post, it seems to come down to supporting `content://` URI.
